### PR TITLE
MGMT-11550: Remove unit test duplicates - Update Network

### PR DESCRIPTION
**Pull request by nmagnezi**
## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/bminventory/inventory_test.go [hunk 1]
```diff
@@ -2702,15 +2702,19 @@ var _ = Describe("cluster", func() {
...
 				Expect(err).ToNot(HaveOccurred())
-				mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(1)
 			})
 
 			mockSuccess := func(times int) {
 				mockClusterUpdateSuccess(times, 3)
 			}
 
+			mockClusterUpdatability := func(times int) {
+				mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(times * 1)
```
Comment (vrutkovs) [inline internal/bminventory/inventory_test.go:2712]: why not
```suggestion
				mockClusterApi.EXPECT().VerifyClusterUpdatability(gomock.Any()).Return(nil).Times(times)
```
?

File: internal/bminventory/inventory_test.go [hunk 2]
```diff
@@ -3625,7 +3806,7 @@ var _ = Describe("cluster", func() {
 					verifyApiErrorString(reply, http.StatusBadRequest, "Machine network CIDR '': Failed to parse CIDR '': invalid CIDR address: ")
 				})
 
-				It("Override networks with new API", func() {
+				It("Override networks", func() {
```
Comment (vrutkovs) [inline internal/bminventory/inventory_test.go:3809]:
```suggestion
				It("Override networks - additional subnet", func() {
```

File: internal/bminventory/inventory_test.go [hunk 3]
```diff
@@ -3647,7 +3828,7 @@ var _ = Describe("cluster", func() {
 					validateHostsRequestedHostname(actual.Payload)
 				})
 
-				It("Override networks with new API 2", func() {
+				It("Override networks 2", func() {
```
Comment (vrutkovs) [inline internal/bminventory/inventory_test.go:3831]:
```suggestion
				It("Override networks  - 2 additional subnets", func() {
```

**Review by vrutkovs — COMMENTED** [review]
lgtm, few cosmetic changes

File: internal/bminventory/inventory_test.go (hunk 1 above)
Comment (nmagnezi) [reply internal/bminventory/inventory_test.go:2712]: Copied over from:
https://github.com/openshift/assisted-service/blob/2650b97301937a647c2a26e9643faa8499d26b23/internal/bminventory/inventory_test.go#L4832-L4834

Will change it as you proposed 👍

Comment (nmagnezi) [reply internal/bminventory/inventory_test.go]: Done.

File: internal/bminventory/inventory_test.go (hunk 3 above)
Comment (nmagnezi) [reply internal/bminventory/inventory_test.go]: Done.

File: internal/bminventory/inventory_test.go (hunk 2 above)
Comment (nmagnezi) [reply internal/bminventory/inventory_test.go]: Done.

**Pull request merged**